keydb.conf

```diff
@@ -1881,6 +1881,14 @@ jemalloc-bg-thread yes
 #  Tuning this parameter is a tradeoff between locking overhead and distributing the workload over multiple cores
 # min-clients-per-thread 50
 
+# How often to run RDB load progress callback?
+# The callback runs during key load to ping other servers and prevent timeouts.
+# It also updates load time estimates.
+# Change these values to run it more or less often. It will run when either condition is true.
+# Either when x bytes have been processed, or when x keys have been loaded.
+# loading-process-events-interval-bytes 2097152
+# loading-process-events-interval-keys 50
```

Comment:
Why is this 50 when the default in the code is 8192?

Reply:
Whoops, that was there from my own testing and I completely missed it when I removed the inline comments.